core/src/main/scala/scala/pickling/pickler/Nothing.scala

```diff
@@ -0,0 +1,16 @@
+package scala.pickling.pickler
+
+import scala.pickling._
+import PicklingErrors.NothingIsNotUnpicklable
+
+trait NothingPicklers {
+  implicit object NothingPicklerUnpickler extends AbstractPicklerUnpickler[Nothing] {
+    override def tag: FastTypeTag[Nothing] = FastTypeTag.Nothing
+    /** Impossible to call in Scala, no value can be typed [[Nothing]] */
+    override def pickle(picklee: Nothing, builder: PBuilder): Unit = ???
```

Comment:
What do you mean? I had this design before, but I realised it's not possible to call this method, actually, because no value can be typed Nothing and passed as an argument.